docs/user-guide/concepts.md

```diff
@@ -531,7 +531,7 @@ We are considering replacing this with the standard
 
 #### hnc.x-k8s.io/excluded-namespace (label on namespaces)
 
-***Excluded namespaces configuration is only available in HNC v0.8 and later***
+***This label is only used in HNC v0.8 (deprecated in HNC v0.9 and later)***
```

Comment:
"Deprecated" means "it's still there and it still works but you shouldn't use it anymore because it will be removed in the future." I think we should either call this "removed" or "ignored" (if that's correct).

Reply:
using `not applicable` suggested below.